main.go

```diff
@@ -34,6 +34,15 @@ type DatsetAPIStore struct {
 	*graph.DB
 }
 
+type initialisedStruct struct {
+	generateDownloadsProducer bool
+	auditProducer             bool
+	mongo                     bool
+	healthTicker              bool
+}
+
+var initialised = initialisedStruct{true, true, true, true}
```

Comment:
Minor comment but as a general rule its probably best practice to use the field names when initialising a struct especially when they are all of the same type.
It makes it clear what fields are being set but also and if we ever added another field here it would be unclear which 4 fields are being initialised and which one isn't.